ui/src/app/pages/datasets/details/dataset.details.component.html

```diff
@@ -25,14 +25,19 @@
       <div class="col-6 d-flex flex-column align-items-start">
         <p class="detail-title">Agent Group Name<span class="ns1red">*</span></p>
         <p>{{ agentGroup?.name }}</p>
-        <p *ngIf="errors?.agentGroup" class="detail-title">{{ errors.agentGroup }}<span class="ns1red">
-          {{ this.dataset.agent_group_id }}</span></p>
+        <p *ngIf="errors?.agentGroup" class="detail-title">{{ errors.agentGroup }}
+          <span class="ns1red">{{ this.dataset.agent_group_id }}</span>
```

Comment:
Is possible display the name instead of the ID? I guess use never have access to the ID on ORB UI

Reply:
@manrodrigues I don't think so, since the entity has already been deleted from the database, hence the error `unable to retrieve`.
The dataset only keeps reference to the agent group is `id`.
The required information no longer exists.